### PR TITLE
[TM first] Light Step Changes & New SPECIAL

### PR DESCRIPTION
**Pull request by Vizoplez**
## About The Pull Request
This was literally just /tg/ light step. It hadn't even been recoded to interact with any of our new systems, including the FOV cone.

As it stands; this now also adds a new SPECIAL - 'Underworld Assassin'
-Takes -1 STR
-Gives 2 END
-Gives 2 SPD
-Gives 2 SNEAKING
-Gives 1 CLIMBING & ATHLETICS
-Light step

This now goes ahead and makes Light Step into Lightfooted, a rarer version (with the original removed) that does three primary things;

-Entirely Nullifies being able to see people moving behind you when they have this trait
-Removes the sound of their footsteps
-Removes the sound of their armor clanking (This is likely going to be nerfed if its _too good_.
+1 SPEED and ATHLETICS

## Why It's Good For The Game
This was one of the most dogshit additions to the SPECIAL PR. Ever. (involving the original addition of the light step trait). This hopefully rounds it out, and makes it better.

It was a trait that straight up just used /tg/ content with 0 changes, 0 additions, 0 care.

This makes it actually do something.

### CODE REVIEW

**Comment by Nakalon** [comment]
Since this touches balance I will let other people weigh in!

**Comment by atomicplan666** [comment]
Based PR

File: code/datums/special_traits/traits/traits.dm [hunk 1]
```diff
@@ -54,12 +54,13 @@
 /datum/special_trait/value/on_apply(mob/living/carbon/human/character, silent)
 	ADD_TRAIT(character, TRAIT_SEEPRICES, "[type]")
 
-/datum/special_trait/lightstep
-	name = "Light Step"
-	greet_text = span_notice("I am quiet, nobody can hear my steps.")
-	weight = 100
+/datum/special_trait/silentstep
+	name = "Silent Step"
+	greet_text = span_notice("For better or for worse; you have been blessed by the Gods. Your footsteps are entirely silent; and armour is unusually quiet on your frame...")
```
Comment (Gyrandola) [inline code/datums/special_traits/traits/traits.dm:59]: I dont like it being a gods thing.

**Review by Nakalon — APPROVED** [review]
Everything looks good.
I personally like the idea of the assassin special as it may lead to conflict which is always good.
However, due to balance concerns I will do the nuke SOP and ask Gyran or Zirok to merge if they agree with this.

**Comment by Squird5** [comment]
I'd have the armor rattling suppression behind a sneak intent check

**Pull request merged**